src/ray/raylet/raylet_client.h

```diff
@@ -152,13 +152,13 @@ class RayletClient {
 
   JobID GetDriverID() const { return driver_id_; }
 
-  bool IsWorker() const { return is_worker_; }
+  bool IsWorker() const { return client_type_ == ClientType::WORKER; }
```

Comment:
Let's change this to `GetType`. Because otherwise the caller may think the client is a driver when `IsWorker` returns False.